src/components/evaluation/EvaluationTable.vue

```diff
@@ -532,16 +535,32 @@ export default {
     getEvaluationTypes().then(data => {
       this.evaluationTypes = [{id: null, name: 'None'}].concat(data)
     })
-    this.rules.currentTermDate = v => {
-      const formatted = this.$moment(v).format('YYYY-MM-DD')
-      if (formatted > this.$config.currentTermDates.begin && formatted < this.$config.currentTermDates.end) {
-        return true
+    this.rules.currentTermDate = (selectedDate, sisStartDate, sisEndDate) => {
+      const formattedSelectedDate = this.$moment(selectedDate).format('YYYY-MM-DD')
+      const formattedCourseStart = this.$moment(sisStartDate).format('YYYY-MM-DD')
+      const formattedCourseEnd = this.$moment(sisEndDate).format('YYYY-MM-DD')
+
+      const courseLength = this.$moment(formattedCourseEnd).diff(formattedCourseStart, 'days')
+      let lastStartDate =  this.$moment(sisEndDate)
+
+      if (this.selectedTerm.name.includes('Summer')) {
+          lastStartDate = courseLength < 90 ? lastStartDate.subtract(13, 'day') : lastStartDate.subtract(20, 'day')
+          return formattedSelectedDate <= lastStartDate.format('YYYY-MM-DD') ? true : 'Date must be within current term.'
+      } else {
+        
+        if (formattedCourseEnd <= this.$config.currentTermDates.end) {
```

Comment:
@cesarvh Code in this method probably needs to change to reference the new `defaultTermDates`?